source/slang/slang-lower-to-ir.cpp

```diff
@@ -10980,13 +10980,13 @@ RefPtr<IRModule> generateIRForTranslationUnit(
         //
         eliminateDeadCode(module, dceOptions);
 
+        if (stripOptions.stripSourceLocs && linkage->m_optionSet.shouldHaveSourceMap())
```

Comment:
Isn't this `linkage->m_optionSet.shouldObfuscateCode() && linkage->m_optionSet.shouldHaveSourceMap()`?

Reply:
I updated the patch.

Reply:
what is the difference between `shouldObfuscateCode` and `shouldStripNameHints`?